### DISCUSSION TITLE
Configurable "Almost Maximise"

### DISCUSSION THREAD

**Discussion by vikulikov** [rxhanson/Rectangle#1351] [Ideas]
Allow to configure the "almost" aspect of the command to be able to set it as percent of height and width of the screen

**Comment by Aaron-Rumpler** [comment]
[Rectangle already supports this](https://github.com/rxhanson/Rectangle/blob/main/TerminalCommands.md#almost-maximize), though it's not exposed in the GUI. You can run the following in the terminal to configure the size:
```sh
defaults write com.knollsoft.Rectangle almostMaximizeHeight -float <VALUE_BETWEEN_0_&_1>
defaults write com.knollsoft.Rectangle almostMaximizeWidth -float <VALUE_BETWEEN_0_&_1>
```

**Reply by akoskohl** [reply]
thanks a lot for both the question and the answer.
